app/queries/get_content_collection.rb

```diff
@@ -11,14 +11,13 @@ def initialize(content_format:, fields:, publishing_app: nil, locale: nil)
 
     def call
       validate_fields!
-
       content_items.map do |content_item|
         hash = Presenters::Queries::ContentItemPresenter.new(
```

Comment:
I think we should rename the `hash` variable to `presenter`.

Alternatively, we could call chain a call to `#present` onto the presenter, e.g.

```ruby
hash = Presenters::Queries::ContentItemPresenter.new( ... ).present
```